services/QuillLMS/app/controllers/cms/cms_controller.rb

```diff
@@ -15,7 +15,7 @@ class Cms::CmsController < ApplicationController
     @subscription_payment_methods = Subscription::PAYMENT_METHODS.dup
     # we do not want credit card here as it should only occur automatically
     @subscription_payment_methods.delete('Credit Card')
-    @promo_expiration_date = Subscription.promotional_dates[:expiration]
+    @promo_expiration_date = Date.today + 1.year
```

Comment:
Shoulud we get rid of `@promo_expiration_date` too and also references to `promoExpiration` (e.g. https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/client/app/bundles/Teacher/containers/EditOrCreateSubscription.jsx#L257)